README.md

```
@@ -18,4 +19,5 @@
 [DI-1094]: https://cuhbioinformatics.atlassian.net/browse/DI-1094
 [DI-1189]: https://cuhbioinformatics.atlassian.net/browse/DI-1189
 [DI-435]: https://cuhbioinformatics.atlassian.net/browse/DI-435
-[DI-1294]: https://cuhbioinformatics.atlassian.net/issues/DI-1294
+[DI-1294]: https://cuhbioinformatics.atlassian.net/issues/DI-1294
+[DI-1299]: https://cuhbioinformatics.atlassian.net/browse/DI-1299
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)